restaurants_scraper.py

```diff
@@ -91,7 +93,12 @@ def get_restaurant_info(restaurant_tag):
         restaurant_html = get_html_and_parse(restaurant_url)
         restaurant_data = restaurant_html.findAll("script")[1].contents[0]
         restaurant_data = ast.literal_eval(restaurant_data)
-        restaurant_information = _restaurant_info(restaurant_data)
+        try:
+            mailA = restaurant_html.findAll('a', href=re.compile("mailto"))
+            em = mailA[0]['href'].split('?')[0][7:]
```

Comment:
```suggestion
            em = mailA[0]['href'].split('?')[0][len("mailto") + 1:]
```
Or maybe even have "mailto" in a variable?